static/js/components/tiles/scatter_tile.tsx

```diff
@@ -155,18 +156,29 @@ export function ScatterTile(props: ScatterTilePropType): JSX.Element {
       replacementStrings={getReplacementStrings(props, scatterChartData)}
       className={`${props.className} scatter-chart`}
       allowEmbed={true}
-      getDataCsv={
-        scatterChartData
-          ? () =>
-              scatterDataToCsv(
-                scatterChartData.xStatVar.statVar,
-                scatterChartData.xStatVar.denom,
-                scatterChartData.yStatVar.statVar,
-                scatterChartData.yStatVar.denom,
-                scatterChartData.points
-              )
-          : null
-      }
+      getDataCsv={() => {
+        // Assume both variables will have the same date
+        const date =
+          props.statVarSpec.length > 0 ? props.statVarSpec[0].date : undefined;
```

Comment:
What would it take to build support for fetching variables with two different dates? This assumption breaks for charts like the top of https://datacommons.org/explore/#q=housing%20vs%20obesity%20in%20US%20states

Reply:
I think this assumption technically [still holds](https://github.com/datacommonsorg/website/blob/ead9e365979eb0e4f66f4981b8103e55b7a2359a/server/lib/nl/config_builder/scatter.py#L51) with the scatter plot -in this case the date is just set to `"LATEST"` or `""`

Granted I'd still like to make getCsv a little more flexible and support more of a "stat var spec" style interface - i  can do that in a follow-up.